packages/cli/src/utils/api.js

```diff
@@ -13,11 +13,13 @@ const AdmZip = require('adm-zip');
 const fetch = require('node-fetch');
 const path = require('path');
 
-const { writeFile, readFile } = require('./files');
+const { writeFile, readFile, ensureDir } = require('./files');
 
 const { prettyJSONstringify, startSpinner, endSpinner } = require('./display');
 
 const { localAppCommand } = require('./local');
+const { promisify } = require('./promisify');
+const { pipeline } = require('stream');
```

Comment:
Node.js v15+ has builtin [Streams Promises API](https://nodejs.org/api/stream.html#streams-promises-api), so we don't need `./promisify`.

```suggestion
const { pipeline } = require('stream/promises');
```